pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -144,3 +193,18 @@ func RemoveLoggingCredentials(cluster *capi.Cluster, loggingCredentials *v1.Secr
 
 	return secretUpdated
 }
+
+// RemoveTracingCredentials - Remove credentials from TracingCredentials secret
+func RemoveTracingCredentials(cluster *capi.Cluster, tracingCredentials *v1.Secret) bool {
```

Comment:
Same for RemoveTracingCredentials and RemoveLoggingCredentials as the secret is provided anyway